docs/.vuepress/theme/styles/layout-center.styl

```diff
@@ -32,3 +32,9 @@
 		padding-top: 1em;
 	}
 }
+
+// now the main.page element is capping the width
```

Comment:
@jessicaschilling I added this so that the back to top arrow aligns with the content easily.
unless you added the max-width on both of these elements to reduce content width for readability?

Reply:
Wasn't me!

Reply:
### Before

![nft-school-staging on fleek co_(Laptop with HiDPI screen)](https://user-images.githubusercontent.com/2353186/119174339-5fb11680-ba60-11eb-9980-a0b7a1468544.png)

### After

![bafybeibweww2oh2yysuj4z7yzkrsgjflkama7lr5zxieakhd7lzw5p7ena on fleek co_(Laptop with HiDPI screen)](https://user-images.githubusercontent.com/2353186/119174356-6475ca80-ba60-11eb-985f-e276a672e53a.png)

Reply:
I think that's fine 😊

Reply:
Will close then 👍